TODO

```diff
@@ -0,0 +1,13 @@
+I already see young incompetent programmers(much more incompetent than I was when I was in my early 20s),
+grifter 'AI-experts' or AI-focused hypesters, or simply average programmers making sub-100k salary,
+thinking they are doing well against the upcoming massive inflation.
+
+
+Also going forward in 2025+, if you don't use emacs or neovim or any other terminal based text editor,
+you're not a professional. If you can't do software development, write automated tests and call yourself a data
+scientist and/or AI specialist, you are a grifter.
+
+If you make less than 100k and a professional software engineer, it is also a time to ask for a raise from your boss
+or clients. Given the upcoming MASSIVE inflation worldwide, I do not even entertain job offers less than 250k USD.
+
+Until people get Doxx-ed, people should enjoy their extremely valuable privacy.
```

Comment:
okay, I don't think these are relevant here, probably

Reply:
Wooah I'm soo sorry these files are checkedin :sweat_smile: fixing it now!